tests/unit/apps/test_core.py

```diff
@@ -342,7 +342,7 @@ def test_upgrade_plan_ussuri_to_victoria(model):
             coro=model.upgrade_charm(app.name, "ussuri/stable", switch=None),
         ),
         UpgradeStep(
-            description=f"Change charm config of '{app.name}' 'action-managed-upgrade' to False.",
+            description="Change 'action-managed-upgrade' config option to False.",
```

Comment:
I would slightly prefer to keep the `app.name` in the description for consistency with the other steps, but it's not a blocker

Reply:
I was thinking that we could remove name from all steps.
Current plan string:
```bash
Upgrade plan for 'nova-compute' to victoria
    Upgrade software packages of 'nova-compute' from the current APT repositories
        Upgrade software packages on unit nova-compute/0
        Upgrade software packages on unit nova-compute/1
        Upgrade software packages on unit nova-compute/2
    Refresh 'nova-compute' to the latest revision of 'ussuri/stable'
    Change charm config of 'nova-compute' 'action-managed-upgrade' to True.
    Upgrade 'nova-compute' to the new channel: 'victoria/stable'
    Change charm config of 'nova-compute' 'source' to 'cloud:focal-victoria'
```
what I think we should have (shorter lines, which will not creat line break):
```bash
Upgrade plan for 'nova-compute' to victoria
    Upgrade software packages from the current APT repositories
        Upgrade software packages on unit nova-compute/0
        Upgrade software packages on unit nova-compute/1
        Upgrade software packages on unit nova-compute/2
    Refresh to the latest revision of 'ussuri/stable' channel
    Change 'action-managed-upgrade' config option to True.
    Upgrade channel to the 'victoria/stable'
    Change 'source' config option to 'cloud:focal-victoria'
```
What do you say?

Reply:
Now I realized that it will cause an issue in hypervisor upgrade plan, where we do not have the app placeholder, e.g. `Upgrade plan for 'nova-compute' to victoria`. 
Maybe we can add such placeholder to hypervisor planner and remove app name from steps.

Reply:
either is fine as long we're sure we're always printing the first line (`Upgrade plan for 'nova-compute' to victoria`)